app/views/find/courses/_about_course.html.erb

```diff
@@ -1,7 +1,9 @@
 <div class="govuk-!-margin-bottom-8">
   <h2 class="govuk-heading-l" id="section-about">Course summary</h2>
   <div data-qa="course__about_course">
-    <% if course.about_course.present? %>
+    <% if course.published_about_course.present? %>
+      <%= markdown(course.published_about_course) %>
+    <% elsif course.about_course.present? %>
```

Comment:
Why do we check for both? To support preview?

Reply:
that's right!